todo_app/data/db_items.py

```diff
@@ -0,0 +1,61 @@
+import pymongo
+import os
+from todo_app.data.Item import Item
+from bson.objectid import ObjectId
+
+#from todo_app.data.Item import Item
```

Comment:
If you don't expect to use this code in the future, removing it will make the code easier to read